### ISSUE TITLE
[2024.09.04] ndt_scan_matcher fails to build

### ISSUE THREAD

**Issue by esteve** [autowarefoundation/autoware-deb-packages#96]
CMake can't seem to find the `OpenMP::OpenMP_CXX` target:

```console
2024-09-05T02:58:23.8351351Z CMake Error at /opt/ros/humble/share/ament_cmake_auto/cmake/ament_auto_add_library.cmake:66 (add_library):
2024-09-05T02:58:23.8352566Z   Target "ndt_scan_matcher" links to target "OpenMP::OpenMP_CXX" but the
2024-09-05T02:58:23.8353445Z   target was not found.  Perhaps a find_package() call is missing for an
2024-09-05T02:58:23.8354168Z   IMPORTED target, or an ALIAS target is missing?
2024-09-05T02:58:23.8364491Z Call Stack (most recent call first):
2024-09-05T02:58:23.8365142Z   CMakeLists.txt:28 (ament_auto_add_library)
2024-09-05T02:58:23.8365513Z
2024-09-05T02:58:23.8365522Z
2024-09-05T02:58:23.8374791Z CMake Error at /opt/ros/humble/share/ament_cmake_gtest/cmake/ament_add_gtest_executable.cmake:50 (add_executable):
2024-09-05T02:58:23.8376822Z   Target "standard_sequence_for_initial_pose_estimation" links to target
2024-09-05T02:58:23.8378554Z   "OpenMP::OpenMP_CXX" but the target was not found.  Perhaps a
2024-09-05T02:58:23.8380196Z   find_package() call is missing for an IMPORTED target, or an ALIAS target
2024-09-05T02:58:23.8381522Z   is missing?
2024-09-05T02:58:23.8381864Z Call Stack (most recent call first):
2024-09-05T02:58:23.8382823Z   /opt/ros/humble/share/ament_cmake_gtest/cmake/ament_add_gtest_executable.cmake:37 (_ament_add_gtest_executable)
2024-09-05T02:58:23.8384204Z   /opt/ros/humble/share/ament_cmake_auto/cmake/ament_auto_add_gtest.cmake:71 (ament_add_gtest_executable)
2024-09-05T02:58:23.8385131Z   CMakeLists.txt:50 (ament_auto_add_gtest)
2024-09-05T02:58:23.8385482Z
2024-09-05T02:58:23.8385489Z
2024-09-05T02:58:23.8386159Z CMake Error at /opt/ros/humble/share/ament_cmake_gtest/cmake/ament_add_gtest_executable.cmake:50 (add_executable):
2024-09-05T02:58:23.8387316Z   Target "once_initialize_at_out_of_map_then_initialize_correctly" links to
2024-09-05T02:58:23.8388410Z   target "OpenMP::OpenMP_CXX" but the target was not found.  Perhaps a
2024-09-05T02:58:23.8389365Z   find_package() call is missing for an IMPORTED target, or an ALIAS target
2024-09-05T02:58:23.8390021Z   is missing?
2024-09-05T02:58:23.8390351Z Call Stack (most recent call first):
2024-09-05T02:58:23.8391302Z   /opt/ros/humble/share/ament_cmake_gtest/cmake/ament_add_gtest_executable.cmake:37 (_ament_add_gtest_executable)
2024-09-05T02:58:23.8394487Z   /opt/ros/humble/share/ament_cmake_auto/cmake/ament_auto_add_gtest.cmake:71 (ament_add_gtest_executable)
2024-09-05T02:58:23.8395420Z   CMakeLists.txt:53 (ament_auto_add_gtest)
```

**Comment by esteve** [comment]
The root cause for this issue is that the `ndt_omp` package in Autoware was forked from https://github.com/koide3/ndt_omp, which exists in the ROS package index (see https://index.ros.org/p/ndt_omp/github-koide3-ndt_omp/#humble), but hasn't been renamed to include the `autoware_` prefix yet. Therefore, when our build farm installs the dependencies for `ndt_scan_matcher`, `rosdep` installs the version of `ndt_omp` from the ROS repositories, instead of the version that was built by locally.

**Comment by esteve** [comment]
@SakodaShintaro @YamatoAndo please enable the issues tab for https://github.com/tier4/ndt_omp so I can track this issue there. Thanks.

![image](https://github.com/user-attachments/assets/20955064-d320-4a96-a07b-2192443e87cb)

**Comment by SakodaShintaro** [comment]
I'm sorry, I couldn't find that setting. It might be because I don't have the necessary permissions.

![Screenshot from 2024-09-12 09-58-04](https://github.com/user-attachments/assets/bca26abe-add3-421a-bf75-90e1f598190f)
![Screenshot from 2024-09-12 09-58-47](https://github.com/user-attachments/assets/9205facd-5900-4e1f-81e7-38bb57da53b3)

@YamatoAndo 
Could you change the setting?

**Comment by YamatoAndo** [comment]
@esteve Hi, I have enabled the issues tab. Thank you.

**Comment by esteve** [comment]
Fixed in #107
